shopfloor/services/location_content_transfer.py

```diff
@@ -356,7 +328,10 @@ def scan_location(self, barcode):  # noqa: C901
                 message=self.msg_store.cannot_move_something_in_picking_type()
             )
 
-        move_lines = self._find_location_move_lines_from_scan_location(location)
+        move_lines = self.search_move_line.search_move_lines(
+            locations=location, match_user=True, enforce_picking_types=False
```

Comment:
We should still respect the picking types of the menu
```suggestion
            locations=location, match_user=True,
```